app/experimenter/experiments/migrations/0174_switch_to_status_next.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 3.2.3 on 2021-06-10 19:14
+
+from django.db import migrations, models
+
+from experimenter.experiments.constants import NimbusConstants
+
+
+def convert_is_end_requested_to_status_next(apps, schema_editor):
+    NimbusExperiment = apps.get_model("experiments", "NimbusExperiment")
+    NimbusExperiment.objects.filter(
+        status=NimbusConstants.Status.LIVE, is_end_requested=True
+    ).update(status_next=NimbusConstants.Status.COMPLETE)
```

Comment:
Good idea.  There's more cases though, aren't there?  

- Anything `DRAFT` that isn't `IDLE` should have `status_next=LIVE`
- Anything `LIVE` that isn't `IDLE` and has `is_end_requested=False` should have `status_next=LIVE`
- Anything `LIVE` that isn't `IDLE` and has `is_end_requested=True` should have `status_next=COMPLETE`

Reply:
Ah yeah, I think I tunnel-visioned on the is_end_requested case, will add these